onnxruntime/core/graph/graph_proto_serializer.cc

```diff
@@ -43,12 +44,33 @@ void GraphViewerToProto(const GraphViewer& graph_view,
   }
 
   if (include_initializer) {
+    std::unordered_set<std::string> current_scope_initializer_set;
+
     auto& initializers = graph_view.GetAllInitializedTensors();
     for (auto& it : initializers) {
       auto* p_initializer = graph_proto.add_initializer();
       *p_initializer = *(it.second);
+      current_scope_initializer_set.insert(it.first);
+    }
+
+
+    // handle outer scope value which is a constant initializer
+    if (include_outer_scope_values) {
```

Comment:
may not related withi this pr: out of scope initializers are not included in out of scope args? why?

Reply:
I think you are right, Initializer in ORT is NodeArg, so outer scope initializers are included in outer scope args. I will remove the "include_outer_scope_values" argument.